plugins/modules/setup.ps1

```diff
@@ -1129,7 +1129,7 @@ $factMeta = @(
             }
 
             $modelMap = @{
-                kvm = @('KVM', 'KVM Server', 'Bochs', 'AHV')
+                kvm = @('KVM', 'KVM Server', 'Bochs', 'AHV', 'CloudStack KVM Hypervisor')
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.